src/lib/utils/thorchain/lp.ts

```diff
@@ -147,6 +148,94 @@ export const getVolume = async (
   return fromThorBaseUnit(volume).times(runePrice).toFixed()
 }
 
+export const get24hSwapChangePercentage = async (
+  assetId: AssetId,
+  runePrice: string,
+  assetPrice: string,
+): Promise<{ volumeChangePercentage: number; feeChangePercentage: number } | null> => {
+  const poolAssetId = assetIdToPoolAssetId({ assetId })
+  const now = Math.floor(Date.now() / 1000)
+  const twentyFourHoursAgo = now - 24 * 60 * 60
+  const fortyEightHoursAgo = now - 2 * 24 * 60 * 60
+
+  const { data: current24hData } = await axios.get<MidgardSwapHistoryResponse>(
+    `${
+      getConfig().REACT_APP_MIDGARD_URL
+    }/history/swaps?pool=${poolAssetId}&from=${twentyFourHoursAgo}&to=${now}`,
```

Comment:
Note this is probably why we have diff. data as THORYield - depending on the data points, and which endpoints we hit, 24h is not the same 24h (same goes for THORYield AFAIK), it may be either 24 hours ago, or "since 00:00" - absolutely fine IMO since we do with the data we get

Reply:
Yeah I think the "since 00:00" approach is a bit rough, as if you check at 00:01 you end up with really weird data showing.